src/NATS.Client.KeyValueStore/NatsKVContext.cs

```diff
@@ -23,13 +23,14 @@ public class NatsKVContext : INatsKVContext
     private static readonly int KvStreamNamePrefixLen = KvStreamNamePrefix.Length;
     private static readonly Regex ValidBucketRegex = new(pattern: @"\A[a-zA-Z0-9_-]+\z", RegexOptions.Compiled);
 
-    private readonly NatsJSContext _context;
-
     /// <summary>
     /// Create a new Key Value Store context
     /// </summary>
     /// <param name="context">JetStream context</param>
-    public NatsKVContext(NatsJSContext context) => _context = context;
+    public NatsKVContext(INatsJSContext context) => Context = context;
+
+    /// <inheritdoc />
```

Comment:
My OCD thanks you for this. Prob don't need to, but could do the rest while you're here. ;)  Might make sense as a separate maintenance PR.

Reply:
Do you mean using the new type constructors (or whatever they're called)?

Reply:
```/// <inheritdoc />```

Reply:
fixed